net/mwan3/files/usr/sbin/mwan3

```diff
@@ -141,13 +141,21 @@ stop()
 
 	for pid in $(pgrep -f "mwan3rtmon"); do
 		kill -TERM "$pid" > /dev/null 2>&1
-		sleep 1
 		kill -KILL "$pid" > /dev/null 2>&1
 	done
 
 	for pid in $(pgrep -f "mwan3track"); do
 		kill -TERM "$pid" > /dev/null 2>&1
-		sleep 1
+		kill -KILL "$pid" > /dev/null 2>&1
+	done
+
+        sleep 1
+
+	for pid in $(pgrep -f "mwan3rtmon"); do
+		kill -KILL "$pid" > /dev/null 2>&1
+	done
+
+	for pid in $(pgrep -f "mwan3track"); do
```

Comment:
Why do we need the `kill -KILL "$pid" `command twice?

Reply:
not needed. thanks for catching this